superset/assets/src/dashboard/util/dashboardLayoutConverter.js

```diff
@@ -88,13 +88,13 @@ function getChartHolder(item) {
     Math.round(size_y / GRID_RATIO * 100 / ROW_HEIGHT),
   );
   if (code !== undefined) {
-    let markdownContent = '';
-    if (slice_name) {
-      markdownContent = `##### **${slice_name}**\n`;
-    }
+    let markdownContent = ' '; // white-space markdown
```

Comment:
I don't quite follow this comment, what do you mean by it? it could be more clear if you say the approach we're using: use code if it exists, else use title if it exists.

Reply:
if `markdownContent` is null or '', we will add markdown sample code into it. i saw a few places user use empty markdown as blank separator.

Reply:
ohhh I see. thanks for the explanation, good catch.